### PR TITLE
zh-cn: sync with en-US version for `console.count()` method

### PR DESCRIPTION
**Pull request by daiwanxing**

### CODE REVIEW

File: files/zh-cn/web/api/console/count/index.md [hunk 1]
```diff
@@ -1,26 +1,40 @@
...
-title: Console.count()
+title: console.count()
 slug: Web/API/console/count
 ---
 
-{{APIRef("Console API")}}{{Non-standard_header}}
+{{APIRef("Console API")}}
 
-输出 count() 被调用的次数。此函数接受一个可选参数 `label`。
+**`console.count()`** 方法会记录 `count()` 被调用的次数。
```
Comment (fwqaaq) [inline files/zh-cn/web/api/console/count/index.md:8]:
```suggestion
**`console.count()`** 方法会记录调用 `count()` 的次数。
```

File: files/zh-cn/web/api/console/count/index.md [hunk 2]
```diff
@@ -1,26 +1,40 @@
...
+
+```js-nolint
+count()
+count(label)
+```
+
+### 参数
+
+- `label` {{Optional_Inline}}
+  - : 如果提供了一个字符串参数，`count()` 会输出特定标签的调用次数。如果没有提供字符串参数，`count()` 方法则会以 default 的标签输出被调用的次数。
```
Comment (fwqaaq) [inline files/zh-cn/web/api/console/count/index.md:22]:
```suggestion
  - : 一个字符串。如果给定，`count()` 会输出带有该标签的调用次数。如果未提供，调用 `count()` 的行为就像是带有“default”标签一样。
```

File: files/zh-cn/web/api/console/count/index.md [hunk 3]
```diff
@@ -1,26 +1,40 @@
 ---
-title: Console.count()
+title: console.count()
```
Comment (yin1999) [inline files/zh-cn/web/api/console/count/index.md:2]:
```suggestion
title: console：count() 方法
```

image

File: files/zh-cn/web/api/console/count/index.md [hunk 4]
```diff
@@ -31,25 +45,25 @@ greet();
...
+"default: 2"
+"default: 3"
+"default: 4"
 ```
 
-注意最后一行的日志输出：在 11 行对 count() 的单独调用是被当作一个独立事件来处理的。
+标签显示为 `default`，因为没有提供明确的标签。
 
-如果把变量 `user` 当作 `label` 参数传给前面调用的 `count()`，把字符串 "alice" 传给后面调用的 `count()`：
+如果我们将 `user` 变量作为 `label` 参数传递给第一次调用 `count()`，并将字符串”alice“传递给第二次调用：
```
Comment (yin1999) [inline files/zh-cn/web/api/console/count/index.md:59]: 有点机翻的味道，再改改

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/api/console/count/index.md [hunk 5]
```diff
@@ -31,25 +45,25 @@ greet();
...
+"default: 2"
+"default: 3"
+"default: 4"
 ```
 
-注意最后一行的日志输出：在 11 行对 count() 的单独调用是被当作一个独立事件来处理的。
+标签显示为 `default`，因为没有提供明确的标签。
 
-如果把变量 `user` 当作 `label` 参数传给前面调用的 `count()`，把字符串 "alice" 传给后面调用的 `count()`：
+如果我们将 `user` 变量作为 `count()` 函数的 `label` 参数传递给第一个调用，并将字符串”alice“作为第二个调用的参数：
```
Comment (yin1999) [inline files/zh-cn/web/api/console/count/index.md:59]:
```suggestion
如果我们将 `user` 变量作为第一次调用 `count()` 函数时的 `label` 参数传入，并将字符串”alice“作为第二次调用的参数：
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**